Indicators/OptionIndicatorBase.cs

```diff
@@ -83,11 +88,21 @@ public abstract class OptionIndicatorBase : IndicatorBase<IndicatorDataPoint>, I
         /// </summary>
         public IndicatorBase<IndicatorDataPoint> Price { get; }
 
+        /// <summary>
+        /// Gets the mirror option price level, for implied volatility
+        /// </summary>
+        protected IndicatorBase<IndicatorDataPoint> _oppositePrice { get; private set; }
```

Comment:
`protected IndicatorBase<IndicatorDataPoint> _oppositePrice` following the pattern of the other indicators, should this probably be public too?